resources/layout/_footer.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+use App\Widget\PerformanceMetrics;
+use Yiisoft\Html\Html;
+
```

Comment:
We need to declare all the variables used i.e.

```php
/* @var \App\LayoutParameters $layoutParameters */
```

Reply:
done.